tests/test_transports.py

```diff
@@ -64,15 +64,15 @@ async def test_httpcore_request():
         transport.add("GET", "https://foo.bar/", content="foobar")
         with httpcore.SyncConnectionPool() as http:
             (status_code, headers, stream, ext) = http.request(
-                method=b"GET", url=(b"https", b"foo.bar", 443, b"/")
+                method=b"GET", url=(b"https", b"foo.bar", None, b"/")
```

Comment:
I think it's good to keep the port at least in one of these transport tests to verify that it's still supported.